evalml/tests/data_checks_tests/test_id_columns_data_check.py

```diff
@@ -69,15 +69,42 @@ def test_id_columns_warning():
     id_cols_check = IDColumnsDataCheck(id_threshold=1.0)
     assert id_cols_check.validate(X) == [
         DataCheckWarning(
-            message="Columns 'Id', 'col_1_id' are 100.0% or more likely to be an ID column",
+            message="Columns 'Id', 'col_3_id' are 100.0% or more likely to be an ID column",
             data_check_name=id_data_check_name,
             message_code=DataCheckMessageCode.HAS_ID_COLUMN,
-            details={"columns": ["Id", "col_1_id"]},
+            details={"columns": ["Id", "col_3_id"]},
             action_options=[
                 DataCheckActionOption(
                     DataCheckActionCode.DROP_COL,
                     data_check_name=id_data_check_name,
-                    metadata={"columns": ["Id", "col_1_id"]},
+                    metadata={"columns": ["Id", "col_3_id"]},
+                ),
+            ],
+        ).to_dict(),
+    ]
+
+    X_dict = {
+        "col_1_id": [0, 1, 2, 3],
+        "col_2": [2, 3, 4, 5],
+        "col_3_id": [1, 1, 2, 3],
+        "Id": [3, 1, 2, 0],
+        "col_5": [0, 0, 1, 2],
+        "col_6": [0.1, 0.2, 0.3, 0.4],
+    }
+
+    X = pd.DataFrame.from_dict(X_dict)
```

Comment:
I think we would expect `col_3_id` to also show up under `DataCheckMessageCode.HAS_ID_COLUMN`.